### PR TITLE
Ext mail pull request

### PR DESCRIPTION
**Pull request by mbiarnes**
forced the script building in $WORKSPACE - so script is there and should be found.
This was tested and worked in https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/custom/job/mbiarnes/job/pullrequest/job/lienzo-tests-pullrequests/ . I hope the same will happen "life" . 
- otherwise we have to roll back this PR again.
(I don't now how to test this in a more realistic environment)

### CODE REVIEW

File: job-dsls/jobs/kie_jenkinsScripts_PR.groovy [hunk 1]
```diff
@@ -21,7 +21,7 @@ cd job-dsls
 def errorSh='''
 touch trace.sh
```
Comment (winklerm) [inline job-dsls/jobs/kie_jenkinsScripts_PR.groovy:22]: Should the directory change to `$WORKSPACE` be here as well?
```suggestion
cd $WORKSPACE
touch trace.sh
```

Comment (mareknovotny) [reply job-dsls/jobs/kie_jenkinsScripts_PR.groovy:22]: that could be the missing piece :)

Comment (mbiarnes) [reply job-dsls/jobs/kie_jenkinsScripts_PR.groovy:22]: you're absolutely right

**Comment by mbiarnes** [comment]
@winklerm changed

File: job-dsls/jobs/kie_jenkinsScripts_PR.groovy [hunk 2]
```diff
@@ -18,14 +18,14 @@ def kieJenkins_PR='''#!/bin/bash -e
 cd job-dsls
 ./gradlew clean test'''
 
-def errorSh='''
+def errorSh='''#!/bin/bash -e
+cd $WORKSAPCE
```
Comment (winklerm) [inline job-dsls/jobs/kie_jenkinsScripts_PR.groovy:22]: Please fix the typo (or accept my suggestion, please :) )

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Comment by mbiarnes** [comment]
Jenkins retest this please

**Pull request merged**